frontend/public/components/utils/async.tsx

```diff
@@ -20,6 +20,7 @@ export class AsyncComponent extends React.Component<AsyncComponentProps, AsyncCo
 
   private retryCount: number = 0;
   private maxRetries: number = 25;
+  private _isMounted: boolean = false;
```

Comment:
isMounted is a React.Component instance method. Thats why we rename

Reply:
In `packages` we disallow `no-underscore-dangle`. I guess in `public` it's allowed :(
I prefer to find a name that doesn't use it so it will be one less change in the future but at this point it's not such a big deal.